app/jobs/dqt/recommend_for_award_job.rb

```diff
@@ -30,7 +30,7 @@ def perform(trainee)
 
     def survey_should_be_scheduled?
       # Don't send survey for Assessment Only routes or EYTS awards
-      !trainee.assessment_only? && trainee.training_route_manager.award_type != EYTS_AWARD_TYPE
+      !(trainee.assessment_only? || trainee.training_route_manager.award_type == EYTS_AWARD_TYPE)
```

Comment:
Looks good, maybe simplify a bit because it's delegated from Trainee.
```suggestion
      !(trainee.assessment_only? || trainee.award_type == EYTS_AWARD_TYPE)
```